app/views/job_invocations/show.html.erb

```diff
@@ -1,7 +1,9 @@
 <% title @job_invocation.description %>
 <% stylesheet 'foreman_remote_execution/foreman_remote_execution' %>
 <% javascript 'charts', 'foreman_remote_execution/template_invocation' %>
-<%= javascript_include_tag *webpack_asset_paths('foreman_remote_execution', :extension => 'js'), "data-turbolinks-track" => true, 'defer' => 'defer' %>
+<% content_for :javascripts do %>
+  <%= javascript_include_tag *webpack_asset_paths('foreman_remote_execution', :extension => 'js') %>
```

Comment:
Maybe a `javascript` can be used for `webpack` assets as well? It should then go in the header as well, right?